lib/functions/since.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Since
+  def self.calculate_units
+    ->(from, to, unit) { get_diff_of_date(from, to, unit) }
+  end
+
+  def self.get_diff_of_date(from, to, unit)
+    valid_date_format?(from, to)
+    valid_unit?(unit)
+    diff = to - from
+    distance = diff / 1.send(unit)
+    distance.abs.round
+  end
+
+  def self.valid_date_format?(from, to)
```

Comment:
you are not using this method in conditions so it should not have `?`. Since you are validating the format call it `validate_date_format!`. `!` at the end means that method may raise an exception